### ISSUE TITLE
Ask about adjusting learning rate

### ISSUE THREAD

**Issue by hankyul2** [zeiss-microscopy/BSConv#6]
Hello!

I read your paper very well and got amazed by your work!

While I am googling about Wide-resnet, I found this [repo](https://github.com/meliketoy/wide-resnet.pytorch) explaining about training best wide-resnet. So how do you think about apply this learning details in your [bin/bsconv_pytorch_train.py](https://github.com/zeiss-microscopy/BSConv/blob/master/bin/bsconv_pytorch_train.py) if and only if it improve results. 

If you think this is good idea, let me know and I will try this.

Thank you.
